samples/meeting-live-coding-interview/csharp/MeetingLiveCoding/MeetingLiveCoding.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <LangVersion>latest</LangVersion>
+    <UserSecretsId>82c81116-057f-40bf-9bf2-2563189614e8</UserSecretsId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="ClientApp\src\components\recruiting-details\**" />
+    <Content Remove="ClientApp\src\components\recruiting-details\**" />
+    <EmbeddedResource Remove="ClientApp\src\components\recruiting-details\**" />
+    <None Remove="ClientApp\src\components\recruiting-details\**" />
+  </ItemGroup>
+
+  <ItemGroup>
```

Comment:
Are the below references (except Microsoft.AspNetCore.SpaServices and Extensions) actually needed? I don't see them being used anywhere